### PR TITLE
Fix responed data type error

### LINKED ISSUE #15: use nginx proxy_pass，when get file  will cause an error
nginx conf 
```
upstream testweb{
        server test.com max_fails=3; 
        keepalive 16;
        }
server {

        listen      443;
        server_name  domain.com;
        ssl on;
        ssl_certificate /usr/local/nginx/ssl/fullchain.key.pem;
        ssl_certificate_key /usr/local/nginx/ssl/privkey.key;
       location / {
	    proxy_pass http://testweb;
      }
```
webdav server 
> test.com

command
> davserver -D D:\webdavserver -n -J -v  -l INFO -H 0.0.0.0

When get webdav file,error info
image

```
Exception happened during processing of request from ('xxx.xxx.xxx.xx', 48906)
Traceback (most recent call last):
  File "X:\cmder\vendor\python3\Lib\socketserver.py", line 639, in process_request_thread
    self.finish_request(request, client_address)
  File "X:\cmder\vendor\python3\Lib\socketserver.py", line 361, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "X:\cmder\vendor\python3\Lib\socketserver.py", line 696, in __init__
    self.handle()
  File "X:\cmder\vendor\python3\Lib\http\server.py", line 420, in handle
    self.handle_one_request()
  File "X:\cmder\vendor\python3\Lib\http\server.py", line 386, in handle_one_request
    self.raw_requestline = self.rfile.readline(65537)
  File "X:\cmder\vendor\python3\Lib\socket.py", line 586, in readinto
    return self._sock.recv_into(b)
ConnectionResetError: [WinError 10054]
```

### PR DESCRIPTION
**Pull request by arschlochnop**
Fixes #15

### CODE REVIEW

File: pywebdav/lib/WebDAVServer.py [hunk 1]
```diff
@@ -73,28 +73,31 @@ def send_body(self, DATA, code=None, msg=None, desc=None,
...
+                            output.write(buf)
+                    output.close()
+                    buffer.seek(0)
+                    DATA = buffer.getvalue()
+                    self.send_header('Content-Encoding', 'gzip')
+
+                self.send_header('Content-Length', len(DATA))
+                self.send_header('Content-Type', ctype)
+            except:
+                pass
```
Comment (andrewleech) [inline pywebdav/lib/WebDAVServer.py]: This would be better logged, eg
```python
except Exception as ex:
    log.exception(ex)
```

Comment (andrewleech) [reply pywebdav/lib/WebDAVServer.py]: I made this change during merge ;-)

**Comment by andrewleech** [comment]
Bah, github can't seem to track rebases, this has been merged in b8451c01 thanks!

**Comment by andrewleech** [comment]
Would you mind confirming quickly that master now works for you? 
Then I'll release it to pypi.

**Comment by arschlochnop** [comment]
> Would you mind confirming quickly that master now works for you?
> Then I'll release it to pypi.

yes ，I have tested through

File: pywebdav/lib/WebDAVServer.py (hunk 1 above)
Comment (arschlochnop) [reply pywebdav/lib/WebDAVServer.py]: Good job，The exception of this place has forgotten to be modified.

**Comment by andrewleech** [comment]
Thanks: https://pypi.org/project/PyWebDAV3/0.9.13/

**Pull request closed without merging**